telemetry/main.schema.json

```diff
@@ -561,10 +561,30 @@
               "required": [
                 "scalars"
               ]
+            },
+            "content": {
+              "type": "object",
+              "properties": {
+                "histograms": {
+                  "type": "object",
+                  "additionalProperties": {
+                    "$ref": "#/definitions/histogram"
+                  }
+                },
+                "keyedHistograms": {
+                  "type": "object",
+                  "additionalProperties": {
+                    "additionalProperties": {
+                      "$ref": "#/definitions/histogram"
+                    }
+                  }
+                }
+              }
             }
           },
           "required": [
-            "parent"
+            "parent",
+            "content"
```

Comment:
Yup. See [TelemetrySession around line 1312](https://dxr.mozilla.org/mozilla-central/rev/3f4c3a3cabaf94958834d3a8935adfb4a887942d/toolkit/components/telemetry/TelemetrySession.jsm#1312).

There _may_ have been a short period of time where it could have one but not the other... but I don't think it straddled a merge day.